notebooks/paper-notebook.ipynb

```diff
@@ -265,7 +265,7 @@
     "```\n",
     "where the first three optional arguments specify a strategy for iterating across the batch: Fortran's loop-parallel `do concurrent` construct, OpenMP multithreading, or an `elemental` procedure that operates on whole `tensor_t` arrays or array slices in one `infer` invocation.  If none of the first three optional arguments exists on the command line, then all three execute.  The fourth optional argument decides whether to additionally perform inference using double precision. The final optional argument determines the number of times each strategy will execute.  See {cite}`rouson2025automatically` for a discussion of the performance of each of these approaches.\n",
     "\n",
-    "Before running `concurrent-inferences`, download the pretrained aerosol model file [model.json](./assets/model.json) and save it to the root directory of your Fiats clone. \n",
+    "Before running `concurrent-inferences`, download the pretrained aerosol model file [model.json](https://github.com/ucar-sea/SEA-ISS-2025-Cloud-microphysics-training/blob/main/assets/model.json) and save it to the root directory of your Fiats clone. \n",
```

Comment:
```suggestion
    "Before running `concurrent-inferences`, download the pretrained aerosol model file {download}`model.json <../assets/model.json>` and save it to the root directory of your Fiats clone. \n",
```

Alright, I think this should do what you'd hoped (i.e. include the file in the build and create a download link in the rendered notebook) rather than needing to link to the file in the repository with the full URL.

The original link syntax with the relative path wasn't working because it's not interpreted as a source or image file.